sklearn/preprocessing/tests/test_imputation.py

```diff
@@ -358,3 +358,35 @@ def test_imputation_copy():
 
     # Note: If X is sparse and if missing_values=0, then a (dense) copy of X is
     # made, even if copy=False.
+
+
+def test_imputed_features():
+    X = np.arange(20,dtype=float).reshape(5,4)
+    n_samples, n_features = X.shape
+    X[0][0] = np.nan
+    X[1][3] = np.nan
+    X[3][2] = np.nan
+    X[:, 1] = np.nan
+
+    imputer = Imputer(missing_values=np.nan, strategy='mean',axis=0)
```

Comment:
pep8 space after comma. Could you run flake8 on your code at all times. Also note that when doing so, you are expected to fix pep8 inconsistencies only for the code that you edit/add.